src/libostree/ostree-sign-ed25519.c

```diff
@@ -487,12 +487,16 @@ _load_pk_from_stream (OstreeSign *self,
   while (TRUE)
     {
       gsize len = 0;
-      g_autofree char *line = g_data_input_stream_read_line (key_data_in, &len, NULL, error);
+      g_autoptr(GError) local_error = NULL;
+      g_autofree char *line = g_data_input_stream_read_line (key_data_in, &len, NULL, &local_error);
       g_autoptr (GVariant) pk = NULL;
       gboolean added = FALSE;
 
-      if (*error != NULL)
-        return FALSE;
+      if (local_error != NULL)
```

Comment:
Minor: the ordering of this feels slightly weird, I'd suggest to reorder variable declarations so that this error-check comes right after the corresponding `g_data_input_stream_read_line` call.

Reply:
Done!